manager/integration/scripts/flake8-check.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash 
+
+source manager/integration/.venv/bin/activate 
+
+pip install -r manager/integration/tests/requirements.txt
+
+flake8 manager/integration/tests
```

Comment:
flake8 has no output, add `echo flake8 check succeed` at the end?

Reply:
Done